src/Adapter/Routing/AdminLinkBuilder.php

```diff
@@ -90,12 +90,13 @@ public function getEditLink($entity, array $parameters)
      */
     private function buildActionParameters($action, $entity, array $parameters)
     {
+        unset($parameters['current_index']);
```

Comment:
always check if key exist before unset.

Reply:
At least for token

Reply:
I think `unset` doesn't care if the key is present, if it's not here it simply does nothing
But at least the code is simpler

Reply:
Oh you're right :+1:

Reply:
Well, then is it approved? ^^